src/styles/index.scss

```diff
@@ -1,3 +1,89 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+  border: $border-width dotted $color-border;
+  border-radius: 50%;
+
+  &__center {
+    position: absolute;
+    width: $center-diameter;
+    height: $center-diameter;
+    background-color: $color-center;
+    border-radius: 50%;
+    z-index: 1;
+  }
+
+  &__hand {
+    position: absolute;
+
+    &--minutes {
+      width: $minute-hand-width;
+      height: $minute-hand-length;
+      top: 20vmin;
+
+      background-color: $color-minute-hand;
+
+      animation-name: rotate-minutes;
+      animation-duration: 3600s;
+      animation-iteration-count: infinite;
+      animation-timing-function: steps(60);
+      transform-origin: bottom center;
+    }
+
+    &--seconds {
+      width: $second-hand-width;
+      height: $second-hand-length;
+      background-color: $color-second-hand;
+      top: 2vmin;
```

Comment:
Similar to the previous comment, verify if the 'top' property value '2vmin' correctly positions the second hand. Ensure consistency in unit usage throughout the styles.